easybuild/easyconfigs/f/FLANN/FLANN-1.9.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,45 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'FLANN'
+version = '1.9.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/mariusmuja/flann/'
+description = "FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/mariusmuja/flann/archive/']
+sources = ['%(version)s.tar.gz']
```

Comment:
```suggestion
sources = ['%(version)s.tar.gz']
patches = ['cmake-flann-1.8.4.patch']
```

Reply:
I have added 
`patches = ['cmake-flann-1.8.4.patch']`
as suggested

Reply:
@sassy-crick we're almost there. it's better to keep the version in the patch file name, so that we can make a new one in case it doesn't apply anymore in later versions.
so, rename the patch to `FLANN-1.8.4_fix-cmake.patch`
then for `FLANN-1.8.4` add this:
```
patches = ['%(name)s-%(versions)s_fix-cmake.patch']
```
and for `FLAN-1.9.1`:
```
patches = ['%(name)s-1.8.4_fix-cmake.patch']
```